priv/static/assets/js/arizona.js

```diff
@@ -107,10 +106,12 @@ globalThis["arizona"] = (() => {
 
   function sendMsgToWorker(event, payload, callback, opts = {}) {
     if (!opts.target && this instanceof HTMLElement) {
-      opts.target = this.getAttribute("arz-target");
+      opts.target = this.getAttribute("arizona-target");
     }
+    const target = document.querySelector(opts.target);
+    const arizonaId = target?.getAttribute("arizona-id") || "[0]";
     callback && subscribeOnce(event, callback, opts);
-    worker.postMessage({ target: opts.target || "root", event, payload });
+    worker.postMessage({ target: arizonaId, event, payload });
   }
```

Comment:
`arz-id` was changed to `arizona-id` and `arz-target` to `arizona-target`.

Reply:
We had an issue for such considerations (https://github.com/arizona-framework/arizona/issues/115). I'd prefer it if we did smaller incremental changes, since they'd be easier to follow: in any case, since this is now done, feel free to cram that issue into this pull request too.

Reply:
I saw it, but it's not related to naming these attributes but to creating new ones to improve "patch" performance. It's a future stuff. I'll rename/update its title.